pytest/tests/shardnet/restake.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import shlex
+import random
+import sys
+from rc import pmap
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+import mocknet
+from configured_logger import logger
+
+
+def restart_restaked_script(restaked_url, delay_sec):
+    s = '''
+        killall restaked
```

Comment:
`killall restaked; sleep 1; killall -9 restaked; rm -f /home/ubuntu/restaked`? -9 to make sure it’s killed, `rm` to make sure we don’t get ETXTBSY if killall takes its sweet time before wget opens the file.